apps/emqx_management/src/emqx_mgmt_api_plugins.erl

```diff
@@ -450,6 +450,21 @@ upload_install(post, #{}) ->
             <<"form-data should be `plugin=@packagename-vsn.tar.gz;type=application/x-gzip`">>
     }}.
 
+do_install_package(NameVsn, FileName, Bin) ->
+    case emqx_plugins:is_allowed_installation(NameVsn) of
+        true ->
+            do_install_package(FileName, Bin);
+        false ->
+            Msg = iolist_to_binary([
+                <<"Package is not allowed installation;">>,
+                <<" first allow it to be installed by running:">>,
+                <<" `emqx ctl plugins allow ">>,
```

Comment:
I notified frontend team to add some special handing for 403 returned from this API.
They will maybe render this message in a pop-up, maybe ask  what's the best quote for this message. 
Unsure ` is the best choice

Reply:
without any change, it already displays a readable error message:

![image](https://github.com/user-attachments/assets/5621b9a7-b233-4d56-b61b-c0539e5b19c8)